### ISSUE TITLE
Gif instead an image [Icons]

### ISSUE THREAD

**Issue by elmundomusical2** [rivafarabi/deckboard#380]
It'll be great than we could choose gif in the icon's buttons, instead of only a image. This function is available in Stream Deck ElGato.
(Actually, if you choose a gif, it playback speed is faster than the gif.)

**Comment by YaFou** [comment]
As far I can see, there are not differences

**Comment by YaFou** [comment]
As far as I can see, there is no difference in the speed when I open the GIF in the explorer and Deckboard.
However, in my point of view, there is not necessary to add a feature to choose the playback speed because a GIF is a simple video with an idea and not a long video.

**Comment by elmundomusical2** [comment]
Try this gif please
![Stream_-_On_ANIMATED](https://user-images.githubusercontent.com/58357415/77530588-e20c6d80-6e91-11ea-9a28-c87549d24d35.gif)

**Comment by YaFou** [comment]
I don't have any problem with the playback speed. Here, a simple video I have recorded to show you: [`download the record`](https://github.com/rivafarabi/deckboard/files/4380502/2020-03-25.12-20-16.zip).

**Comment by elmundomusical2** [comment]
Yes, in the pc app it's ok but in the mobile app , the speed it's so fast...

**Comment by YaFou** [comment]
Hum... You have right. After some time, the two GIF are not synchronised. I think it's a bug or perhaps the speed in the source code is not the same than Windows.

**Comment by elmundomusical2** [comment]
What can I do?

**Comment by YaFou** [comment]
I don't know. Maybe @rivafarabi can clarify this problem?

**Comment by elmundomusical2** [comment]
Yes please. @rivafarabi can you help us?
